error-prone-contrib/src/main/java/tech/picnic/errorprone/refasterrules/BugCheckerRules.java

```diff
@@ -67,4 +68,22 @@ String after(String value) {
       return Constants.format(value);
     }
   }
+
+  /** Prefer {@link Name#contentEquals(CharSequence)} over more verbose alternatives. */
+  static final class NameContentEquals {
```

Comment:
We can debate whether `BugCheckerRules` is the best place for this rule, but it's in line with other rule definitions in this class.